models/dataset.go

```diff
@@ -0,0 +1,27 @@
+package models
+
+const (
+	fmtEditions                   = "/datasets/%s/editions/%s"
```

Comment:
shouldnt need these fmt strings anymore?

Reply:
You are right these are not needed